src/main/java/imsam/Main.java

```diff
@@ -63,6 +63,16 @@ public static void setLogLevel(Level logLevel) {
         loggerContext.updateLoggers();
     }
 
+    public static void disableConsoleLogging() {
+        //loggerContext.getConfiguration()
+        //        .getAppender("stdout")
+        //        .stop();
+
+        Map<String, Appender> appenders = loggerContext.getConfiguration()
+                    .getAppenders();
+        appenders.forEach((name, appender) -> System.out.println(name));
```

Comment:
This won't disable console logging, just show the name of each logging appender

Reply:
Oops, didn't mean to commit that. Still working on how this can be done. Comments added now to indicate current non-funtionality.